lib/util.js

```diff
@@ -620,7 +620,76 @@ function create_wordpress_paginator(feed) {
     return _doCreateWordpress(feed);
 }
 
+/**
+ * Default options for {@link utilcleanup_body}
+ *
+ * @property {Array} remove
+ * @property {Array} removeNoText
+ * @property {Object.<string, Array>} removeAttrs
+ * @property {boolean} noComments
+ * @memberof util
+ */
+const CLEANUP_DEFAULTS = {
+    remove: [ 'script', 'noscript', 'style', 'iframe', 'hr', 'br + br', 'center' ],
+    removeNoText: ['p'],
+    removeAttrs: { '*': [ 'style', 'class' ] },
+    noComments: true,
+};
+
+/**
+ * Utility to remove the article body.
+ *
+ * The cleanup should be among the last operations done with the
+ * body. For example, if your ingester tries to make Video assets from
+ * iframes, the cleanup utility removes iframe elements by default.
+ *
+ * @param {Cheerio} body - The Cheerio object that will be cleaned
+ *   up. Is called body because it usually represents an article body.
+ * @param {Object} options - Options to configure the cleanup, by
+ *   default {@link utilCLEANUP_DEFAULTS} is used. You can expand or
```

Comment:
Missing dot here?

Reply:
No, that's on purpose. The link in the documentation doesn't work otherwise.

Reply:
Node...